__init__.py

```diff
@@ -47,6 +47,7 @@ class VolumeSkill(MycroftSkill):
 
     def __init__(self):
         super(VolumeSkill, self).__init__("VolumeSkill")
+        self.skill_control.category = 'system'
```

Comment:
Is it worth throwing this in a try / except block so we can release it onto mycroft-core installs that don't have the state system already?

eg:
```
        try:
            self.skill_control.category = 'system'
        except AttributeError:
            self.log.debug('Installed mycroft-core does not have Skill State functionality')
```

Reply:
Since this is a set operation won't it succeed even if the member doesn't already exist? won't it just add it?

Reply:
Yes, however the AttributeError is raised because the Skill has no `skill_control` attribute

Reply:
Since the plan is for this to get merged after the core changes, I don't think adding the try block makes much sense.

Reply:
Yeah I've backflipped myself. As long as we know these can't get merged into the default branch of each Skill until the functionality exists in mainstream mycroft-core, not just the Mark-2 feature branch